packages/react-app/src/pages/api/bounties/[id].ts

```diff
@@ -16,13 +16,12 @@ export const handler = async (
 	res: NextApiResponse
 ): Promise<void> => {
 
```

Comment:
editKey was legacy schema, we use RBAC for access control instead so I have removed references to it